include/simdjson/dom/parsedjson_iterator.h

```diff
@@ -266,5 +267,6 @@ class [[deprecated("Use the new DOM navigation API instead (see doc/basics.md)")
 };
 
 } // namespace simdjson
+#endif //#if SIMDJSON_ENABLE_DEPRECATED_API
```

Comment:
Remove the #if, I think--generally we don't include that in the #endif matching comment.

Reply:
sure thing!